docs/content/walkthrough-raster/001-hw-quickstart-guide-raster-demo.adoc

```diff
@@ -100,9 +100,14 @@ Add a spatial index.
 
 [source, bash]
 ----
-geowave config addindex -t spatial spatial
+geowave config addindex -t spatial spatial --crs 3857
 ----
 
+[NOTE]
+====
+We are setting a custom CRS here as an example. If you don't set this GeoWave defaults to EPSG:4326.
```

Comment:
can we use the term Coordinate Refeerence System here (it is the quickstart guide after all)